omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -117,6 +117,17 @@
 default['private_chef']['rabbitmq']['env_path'] = "/opt/opscode/bin:/opt/opscode/embedded/bin:/usr/bin:/bin"
 
 ####
+# External RabbitMQ
+####
+default['private_chef']['external-rabbitmq']['enable'] = false
+default['private_chef']['external-rabbitmq']['actions_user'] = 'actions'
+default['private_chef']['external-rabbitmq']['actions_password'] = 'changeme'
+default['private_chef']['external-rabbitmq']['actions_vhost'] = '/analytics'
```

Comment:
Does this fail if this vhost doesn't exist on the external rabbitmq?  If so, what does that failure look like?